incident_viewer_demo/src/LineChart/LineChart.scss

```diff
@@ -24,46 +24,81 @@
     stroke-opacity: 0.5;
   }
   
+  // Incident Styles Variables
+  $good-color: #44BD1D;
+  $bad-color: orangered;
```

Comment:
Maybe we should call it bad-area-color, so that way its a little more clear

Reply:
bad-hover-color would work too. Unless this is more than just the hover. Oh, reading further I see that it is more than the hover. So I agree with Amy!